owncloud/owncloud.py

```diff
@@ -1016,6 +1016,28 @@ def search_users(self, user_name):
 
         raise HTTPResponseError(res)
 
+    def get_users(self):
```

Comment:
@PVince81 what do you think about this :

```python
    def search_users(self, user_name=''):
        """Searches for users via provisioning API.
        If you get back an error 999, then the provisioning API is not enabled.

        :param user_name:  name of user to be searched for
        :returns: list of usernames that contain user_name as substring
        :raises: HTTPResponseError in case an HTTP error status was returned

        """
        res = self._make_ocs_request(
            'GET',
            self.OCS_SERVICE_CLOUD,
            'users?search=' + user_name
        )

        if res.status_code == 200:
            tree = ET.fromstring(res.content)
            users = [x.text for x in tree.findall('data/users/element')]

            return users

        raise HTTPResponseError(res)

    def get_users(self):
        """Get users via provisioning API.
        If you get back an error 999, then the provisioning API is not enabled.

        :returns: list of usernames
        :raises: HTTPResponseError in case an HTTP error status was returned

        """
        return self.search_users()
```